charon/tests/ui/remove-dynamic-checks.out

```diff
@@ -326,17 +339,55 @@ fn test_crate::mix_arith_i32(@1: i32, @2: i32, @3: i32) -> i32
     return
 }
 
+fn test_crate::shl(@1: u32, @2: u32) -> u32
+{
+    let @0: u32; // return
+    let x@1: u32; // arg #1
+    let y@2: u32; // arg #2
+    let @3: u32; // anonymous local
+    let @4: u32; // anonymous local
+
+    @3 := copy (x@1)
+    @4 := copy (y@2)
+    @0 := move (@3) << move (@4)
+    drop @4
+    drop @3
+    return
+}
+
+fn test_crate::shr(@1: u32, @2: u32) -> u32
+{
+    let @0: u32; // return
+    let x@1: u32; // arg #1
+    let y@2: u32; // arg #2
+    let @3: u32; // anonymous local
+    let @4: u32; // anonymous local
+
+    @3 := copy (x@1)
+    @4 := copy (y@2)
+    @0 := move (@3) >> move (@4)
+    drop @4
+    drop @3
+    return
+}
+
 global test_crate::CONST0  {
     let @0: usize; // return
+    let @1: (usize, bool); // anonymous local
 
-    @0 := const (1 : usize) + const (1 : usize)
+    @1 := const (1 : usize) checked.+ const (1 : usize)
+    assert(move ((@1).1) == false)
```

Comment:
As discussed, we won't touch this for now. But could you open an issue to simplify the constants?

Reply:
Do you want to simplify constants eventually?

Reply:
Not necessarily, but it's good if it is documented.

Reply:
I think issues are not appropriate for documentation. Should we have a markdown file somewhere about supported/unsupported features?

Reply:
Done: https://github.com/AeneasVerif/charon/issues/142